app/src/main/res/values/dimens.xml

```diff
@@ -7,6 +7,7 @@
     <dimen name="space_12dp">12dp</dimen>
     <dimen name="space_16dp">16dp</dimen>
     <dimen name="space_24dp">24dp</dimen>
+    <dimen name="space_32dp">32dp</dimen>
```

Comment:
In my opinion, this should be defined as specific name like `splash_logo_padding` because this is only used in splash page.